captain/types/worker.py

```diff
@@ -40,16 +40,16 @@ def __init__(
         self,
         jobset_id: str,
         sys_status: str | None = None,
-        failed_nodes: list[str] = [],
-        running_nodes: str = "",
+        failed_nodes: list[str] | None = None,
+        running_node: str = "",
         dict_item: dict[str, Any] = {},
     ):
         self["jobsetId"] = jobset_id
         if sys_status:
             self["SYSTEM_STATUS"] = sys_status
         self["type"] = "worker_response"
-        self["FAILED_NODES"] = failed_nodes
-        self["RUNNING_NODES"] = running_nodes
+        self["FAILED_NODES"] = failed_nodes or []
+        self["RUNNING_NODE"] = running_node
```

Comment:
fyi planning on supporting multiple node highlighting in the future, so it should be kept to running_nodes

Reply:
Keeping this for now as we have to adjust the front end as well to support multiple nodes running at a time. we can implement this in the near future when needed as a separate task.